regression/validate-trace-xml-schema/validate-xsd/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Everything in the `gradle` directory as well as the `gradlew` and `gradlew.bat` scripts are autogenerated by gradle so the build can work without a user having to have a compatible gradle version locally. So they're meant to be here, but shouldn't be reviewed.